shell/platform/darwin/ios/framework/Source/FlutterTextInputPlugin.mm

```diff
@@ -2675,8 +2675,7 @@ - (void)hideTextInput {
   [self removeEnableFlutterTextInputViewAccessibilityTimer];
   _activeView.accessibilityEnabled = NO;
   [_activeView resignFirstResponder];
-  [_activeView removeFromSuperview];
-  [_inputHider removeFromSuperview];
+  [self cleanUpViewHierarchy:YES clearText:NO delayRemoval:NO];
```

Comment:
Based on the comment [here](https://github.com/flutter/flutter/blob/021b2b36275342ad94a1ef44f9748b1e6153b0a3/engine/src/flutter/shell/platform/darwin/ios/framework/Source/FlutterTextInputPlugin.mm#L2847-L2855) my assumption is that we should be setting `clearText:YES` in this case.

@LongCatIsLooong can you confirm?

Reply:
It's been a while since I looked at this. Is there a quick explanation of what is supposed to happen here? I think this will be called when the framework dismisses the software keyboard?

Reply:
It's not making a lot sense to me right now that `cleanUpViewHierarchy` needs to be called when the framework dismisses the keyboard. A bit explanation as to why the bug is happening (is the framework dismissing the keyboard before the autofill group cleans up the view hierarchy?)